src/map/CustomGridLayer.tsx

```diff
@@ -48,32 +57,74 @@ export default function CustomGridLayer() {
         };
     }, [map]);
 
-    // bfox6 - When geohashes change, update the map with new Rectangles
+    // bfox6 - When geohashes change, update the map with new Polylines
     useEffect(() => {
```

Comment:
This logic doesn't need to be in a `useEffect` hook. Since the code is all synchronous, you can just create the `latLines` and `lonLines` variables directly (no need for `useState`).